src/main/java/seedu/address/logic/parser/ParserUtil.java

```diff
@@ -98,14 +99,14 @@ public static Name parseName(String name) throws ParseException {
      * @throws ParseException if the given {@code phone} is invalid.
      */
     public static Phone parsePhone(String phone) throws ParseException {
-        if (phone != null) {
-            String trimmedPhone = phone.trim();
-            if (!Phone.isValidPhone(trimmedPhone)) {
-                throw new ParseException(Phone.MESSAGE_CONSTRAINTS);
-            }
-            return new Phone(trimmedPhone);
+        if (phone.isBlank()) {
+            return new Phone(null);
+        }
+        String trimmedPhone = phone.trim();
+        if (!Phone.isValidPhone(trimmedPhone)) {
+            throw new ParseException(Phone.MESSAGE_CONSTRAINTS);
```

Comment:
Apparently I tried changing this
If I offload null checking to this method, `phone.isBlank()` causes NullPointerException, so it needs to be `phone == null || phone.isBlank()`